pytest_doctestplus/output_checker.py

```diff
@@ -189,3 +187,9 @@ def output_difference(self, want, got, flags):
         # new-style class.
         return self._original_output_checker.output_difference(
             self, want, got, flags)
+
+
+def isclose(ng, nw, rtol=1e-05, atol=1e-08):
```

Comment:
could we still rely on numpy when it's available, most packages that use this plugin has it anyway (thus the reason this wasn't very high on our todo)?

it's not a strong preference, I leave it with @astrofrog to decide